### DISCUSSION TITLE
Using @apply to react components scss files

### DISCUSSION THREAD

**Discussion by oroalej** [tailwindlabs/tailwindcss#2717] [Help]
Hi,

Is it even possible to use @apply to react component scss files?
I was able to use it on the file that being compiled but not in components scss files. It is working fine in vuejs but not in react.

I really appreciate your response.

Regards,
Alex

**Comment by minimit** [comment]
Hey, you just need to add **postcss-loader** or a postcss compiler to the stack when compiling scss files.

In the same way that people add [autoprefixer to scss](https://medium.com/@realfrancisyan/scss-autoprefixer-with-webpack-9f8456ae751b) **add postcss-loader** on scss **before sass-loader** in your webpack configuration.

```
module: {
    rules: [
      {
        test: /\.scss$/,
        use: [
          { loader: 'style-loader' },
          {
            loader: 'css-loader'
          },
          {
            loader: 'postcss-loader'
          },
          {
            loader: 'sass-loader'
          }
        ]
      }
    ]
  },
```
